Mathlib/LinearAlgebra/StdBasis.lean

```diff
@@ -112,44 +112,41 @@ theorem infᵢ_ker_proj_le_supᵢ_range_stdBasis {I : Finset ι} {J : Set ι} (h
   SetLike.le_def.2
     (by
       intro b hb
-      simp only [mem_infi, mem_ker, proj_apply] at hb
+      simp only [mem_infᵢ, mem_ker, proj_apply] at hb
       rw [←
-        show (∑ i in I, std_basis R φ i (b i)) = b by
+        show (∑ i in I, stdBasis R φ i (b i)) = b by
           ext i
-          rw [Finset.sum_apply, ← std_basis_same R φ i (b i)]
-          refine' Finset.sum_eq_single i (fun j hjI ne => std_basis_ne _ _ _ _ Ne.symm _) _
+          rw [Finset.sum_apply, ← stdBasis_same R φ i (b i)]
+          refine' Finset.sum_eq_single i (fun j hjI ne => stdBasis_ne _ _ _ _ ne.symm _) _
           intro hiI
-          rw [std_basis_same]
+          rw [stdBasis_same]
           exact hb _ ((hu trivial).resolve_left hiI)]
-      exact sum_mem_bsupr fun i hi => mem_range_self (std_basis R φ i) (b i))
+      exact sum_mem_bsupᵢ fun i hi => mem_range_self (stdBasis R φ i) (b i))
 #align linear_map.infi_ker_proj_le_supr_range_std_basis LinearMap.infᵢ_ker_proj_le_supᵢ_range_stdBasis
 
 theorem supᵢ_range_stdBasis_eq_infᵢ_ker_proj {I J : Set ι} (hd : Disjoint I J)
     (hu : Set.univ ⊆ I ∪ J) (hI : Set.Finite I) :
     (⨆ i ∈ I, range (stdBasis R φ i)) = ⨅ i ∈ J, ker (proj i : (∀ i, φ i) →ₗ[R] φ i) := by
-  refine' le_antisymm (supr_range_std_basis_le_infi_ker_proj _ _ _ _ hd) _
-  have : Set.univ ⊆ ↑hI.to_finset ∪ J := by rwa [hI.coe_to_finset]
-  refine' le_trans (infi_ker_proj_le_supr_range_std_basis R φ this) (supᵢ_mono fun i => _)
+  refine' le_antisymm (supᵢ_range_stdBasis_le_infᵢ_ker_proj _ _ _ _ hd) _
+  have : Set.univ ⊆ ↑hI.toFinset ∪ J := by rwa [hI.coe_toFinset]
+  refine' le_trans (infᵢ_ker_proj_le_supᵢ_range_stdBasis R φ this) (supᵢ_mono fun i => _)
   rw [Set.Finite.mem_toFinset]
-  exact le_rfl
 #align linear_map.supr_range_std_basis_eq_infi_ker_proj LinearMap.supᵢ_range_stdBasis_eq_infᵢ_ker_proj
 
 theorem supᵢ_range_stdBasis [Finite ι] : (⨆ i, range (stdBasis R φ i)) = ⊤ := by
   cases nonempty_fintype ι
-  convert top_unique (infi_emptyset.ge.trans <| infi_ker_proj_le_supr_range_std_basis R φ _)
-  ·
-    exact
-      funext fun i =>
-        ((@supᵢ_pos _ _ _ fun h => range <| std_basis R φ i) <| Finset.mem_univ i).symm
+  convert top_unique (infᵢ_emptyset.ge.trans <| infᵢ_ker_proj_le_supᵢ_range_stdBasis R φ _)
```

Comment:
I think `convert` might have syntax for naming the variables it intros now.

Reply:
Not in the documentation. Let me know what it is if you find it.